crates/wiggle/generate/src/types/flags.rs

```diff
@@ -47,7 +48,7 @@ pub(super) fn define_flags(
                 if #repr::from(!#ident::all()) & value != 0 {
                     Err(wiggle::GuestError::InvalidFlagValue(stringify!(#ident)))
                 } else {
-                    Ok(#ident { bits: value })
+                    Ok(#ident::from_bits_truncate(value))
```

Comment:
`from_bits_truncate` silently drops unknown flags; should this use `from_bits` and return an `InvalidFlagValue` if it doesn't succeed?